Sources/StreamVideo/Call.swift

```diff
@@ -71,6 +71,14 @@ public class Call: @unchecked Sendable, WSEventsSubscriber {
                 .removeDuplicates()
                 .sink { [weak self] in self?.didUpdate($0) }
                 .store(in: &self.cancellables)
+
+            self
```

Comment:
maybe we can move this one and the one above into separate methods? (e.g. `subscribeToTranscriptionUpdates`)